cmb2-conditionals.php

```diff
@@ -21,21 +21,36 @@ function cmb2_conditionals_load_actions()
 	define('CMB2_CONDITIONALS_PRIORITY', 99999);
 
 	add_action('admin_init', 'cmb2_conditionals_hook_data_to_save_filtering', CMB2_CONDITIONALS_PRIORITY);
-	add_action('admin_footer', 'cmb2_conditionals_footer', CMB2_CONDITIONALS_PRIORITY);
 }
 
+add_action('admin_enqueue_scripts', 'cmb2_conditionals_footer', CMB2_CONDITIONALS_PRIORITY);
```

Comment:
Hi, It was actually not working inside the function. Probably because the function is called by the 'plugins_loaded' hook.